_quarto.yml

```diff
@@ -1,6 +1,16 @@
+date: today
```

Comment:
I tend to use
```
date: last-modified
```
now so that simply re-rendering doesn't change the date

Reply:
Oh I like that